api/internal/k8sgen/generator/go.mod

```diff
@@ -0,0 +1,3 @@
+module sigs.k8s.io/kustomize/api/internal/k8skust/generator
+
+go 1.15
```

Comment:
This version is a little bit messy in this repo. 1.14, 1.13, 1.15 are using simultaneously.

Reply:
feel free to send out a pr upgrading everything to 1.15.
This was set to 1.15 because I ran `go mod init` with to go on my local machine